psga/s_aureus/bactopia_one.nf

```diff
@@ -91,6 +93,16 @@ with open('hello.txt', 'w') as outfile:
         '{outdir}/annotation-summary.txt'.format(outdir=outdir)
     )
 
+    # TODO find a better way to get the genome assembly name.
+    # This will find the genome assembly directory and copy it's name
+    # Required because the resultfiles.json needs to know the directory names
+    _ ,gcf_name, _ = next(os.walk('local-single-sample/{sample_id}/variants'.format(sample_id=sample_id), topdown=True))
+    genome_assembly_name = gcf_name[0]
+    print("genome_assembly_name: " + genome_assembly_name)
+    with open('genome_assembly_name.txt', 'w') as genome_assembly_name_file:
```

Comment:
why is this ".txt" and not ".json"?

Reply:
Agree, should be .json

Will ignore, deadline is today.